packages/seal-loupe-adapter/composer.json

```
@@ -29,7 +29,7 @@
     "require": {
         "php": "^8.1",
         "cmsig/seal": "^0.9",
-        "loupe/loupe": "^0.11",
+        "loupe/loupe": "^0.12",
```

```suggestion
        "loupe/loupe": "^0.11 || ^0.12",
```

would add it as additional if there is no breaking change.

PS: `composer update` need to be run to update the lock files.

---

I will need to raise the minimum to 0.12 anyway for https://github.com/PHP-CMSIG/search/pull/551 so I'd prefer to do it here not to pollute the PR with useless diffs :)